Makefile

```diff
@@ -198,7 +239,7 @@ run: build ## runs the service locally
 .PHONY: tools
```

Comment:
`make tools`:
This target isn't used by any of our CI/CD workflows, ie, we don't directly call `tools` inside `make build`,`make test`.. etc. This is more of a dev env setup target which I didn't wanna change much.

Reply:
Its for convenience under the same name roof. If you feel it's better to have it in some shell script then go ahead. Don't have strict preferences here.

Reply:
Yeah, I like it that way. I didn't get a chance to do any due diligence with them in this PR - @tinakurian would be nice if you could verify if `make tools` is doing fine :)